hack/fix_blueprint_swagger

```diff
@@ -33,15 +33,34 @@ def rename_blueprint_models(swagger):
         replace_value(swagger, "$ref", oldRef, newRef)
 
 
+def remove_global_security(swagger):
+    if 'securityDefinitions' in swagger:
+        del swagger['securityDefinitions']
+
+
+# See https://jira.eng.vmware.com/browse/TCS-4017 for context
```

Comment:
Should we provide this link in the open source project?

Reply:
I don't see why not, there are already a few references to internal JIRA's, it's hard to keep track of them if we don't document them where there are needed. But if you think there's a better way to document it, just please let me know.